exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerTest.java

```diff
@@ -150,17 +154,19 @@ void fetchFiltered() {
   }
 
   @Test
-  void fetchPrometheusCompressed() {
+  void fetchPrometheusCompressed() throws IOException {
     WebClient client =
         WebClient.builder("http://localhost:" + prometheusServer.getAddress().getPort())
-            .decorator(DecodingClient.newDecorator())
```

Comment:
The behavior of the `DecodingClient` changed in 1.21.0 to [remove](https://github.com/line/armeria/commit/dd49448e73b905d4f630f361d68779459d7e67d9#diff-50286a10a9131442ce3e1ec05f02294adcf000512a28743cc63c8814dd220e02R104-R107) `Content-Encoding` headers if the request was decoded. This makes it impossible to assert `gzip` encoding.

I've adjusted the test to manually request gzip encoding rather than using the decorator in order to maintain the coverage.